### PR TITLE
v2v:add TC 'Convert VMware ESX VM whose name contains percent encodin…

### PR DESCRIPTION
**Pull request by vwu-vera**
Please check bz#1938954

### CODE REVIEW

**Comment by vwu-vera** [comment]
This PR is related to #3503.

**Comment by vwu-vera** [comment]
(1/1) type_specific.io-github-autotest-libvirt.convert_from_file.positive_test.linux.input_mode.vmx.vmx_ssh.percent_encode.output_mode.libvirt:PASS (252.65 s)

**Comment by vwu-vera** [comment]
@xiaodwan Please help to review. Thanks.:)

**Review by chunfuwen — APPROVED** [review]
LGTM

File: v2v/tests/cfg/convert_from_file.cfg [hunk 1]
```diff
@@ -193,6 +193,12 @@
                                     datastore = esx6.7-function
                                     #Skip pre-check by bz#2068992. Will modify once the bug fixed.
                                     skip_virsh_pre_conn = yes
+                                - percent_encode:
```
Comment (chunfuwen) [inline v2v/tests/cfg/convert_from_file.cfg:196]: Does this PR depend on https://github.com/avocado-framework/avocado-vt/pull/3503 ?

Comment (vwu-vera) [reply v2v/tests/cfg/convert_from_file.cfg:196]: Correct. These 2 PRs are for the same new TC.

**Review by xiaodwan — APPROVED** [review]

**Pull request merged**